pkgs/tools/security/bitwarden-cli/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, nodejs }:
+
+stdenv.mkDerivation rec {
+  name = "bitwarden-cli-${version}";
+  version = "v1.6.0";
+
+  src = fetchFromGitHub {
+    owner = "bitwarden";
+    repo = "cli";
+    rev = "${version}";
+    fetchSubmodules = true; 
```

Comment:
Can you double check that `fetchSubmodules` is necessary? Many projects bundle all of the release artifacts properly in their release tarballs, which is always preferable for us, since we can then just grab the flat tarball instead of the git tree.

Reply:
For this project you either have to fetchSubmodules or keep the dotGit files and npm has a command that will use git to fetch them.  The submodule is pointed at a specific tree/rev so it should be consistent.

Reply:
Just to clarify the release tarball does not include the submodule.  It's unfortunate and I'll leave an issue on the repository to see if they will change it for the next release.